crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -470,6 +471,33 @@ fn equal_update_outputs(node: &mut Node) {
     }
 }
 
+fn expand_update_outputs(node: &mut Node) {
+    let shape = if node.inputs.len() == 2 {
+        match &node.inputs[1].value {
+            Some(value) => match value {
+                Data::Int64s(shape) => Some(shape.clone()),
+                _ => panic!("Expand: invalid input types"),
+            },
+            None => None,
+        }
+    } else {
+        node.attrs.get("shape").cloned().map(|v| v.into_i64s())
```

Comment:
This is not applicable for Expand since there is no attribute `shape` the spec: https://onnx.ai/onnx/operators/onnx__Expand.html. So we should remove it.

Reply:
Yes, you're right! I will get rid of it:)